ckanext/odm_nav/lib/odm_nav_helper.py

```diff
@@ -40,12 +40,11 @@
 
 def load_country_specific_menu(country, wpUrl):
   log.info('getting menu for %s',country)
-
   # list of menu endpoints
   if country=='':
-    menu_endpoint= 'http://pp.' + wpUrl + '/wp-json/menus/824'
+    menu_endpoint= 'http://' + wpUrl + '/wp-json/menus/824'
```

Comment:
@acorbi 
you setup the country specific menu by filling in the menu id of c
you can see all menu id's on pp.opendevelopmentmekong.net/wp-json/menus pp-cambodia.open.../wp-json/menus, ...
please make sure that wp-json plugin is enabled, PP wp had some plugins disabled again after some days including wp-json & wpckan